### PR TITLE
Feature/inbox page

### PR DESCRIPTION
**Pull request by DavidZamora35**
Finished UI for patient inbox page. Still requires some use cases for receiving the user's list of therapists and their chat logs.

### CODE REVIEW

**Comment by jesuskcapellan** [comment]
10 commits, nice

**Review by oa279 — DISMISSED** [review]

File: src/app/(user)/inbox/page.tsx
Comment (jesuskcapellan) [inline src/app/(user)/inbox/page.tsx:1]: I think to solve the height issue, you can try editing the `layout.tsx` file for `app/(user)/`. There should be a div that contains the Breadcrumbs component and the SidebarInset component. Just add `flex flex-col` to that div and then `flex-1` to the SidebarInset. Reach out for more.

File: src/app/(user)/inbox/mock-data.ts
Comment (jesuskcapellan) [inline src/app/(user)/inbox/mock-data.ts:1]: Can we change the message up just so it's clearer that it's changing?

File: src/app/(user)/inbox/chat-box.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        if (message.trim() != "") {
+            item.recievedMessages.push(message);
+            setMessage("")
+        }
+    }
+
+    return(
+        <div className="w-[98%] h-full">
+            <div className="flex flex-col h-full border rounded-[8px]">
+                <div className="flex p-6 border-b-2 gap-4">
```
Comment (jesuskcapellan) [inline src/app/(user)/inbox/chat-box.tsx:31]: can we make the border 1 px to keep consistent?

File: public/demo.png
Comment (jesuskcapellan) [inline public/demo.png:1]: mogged.

File: src/app/(user)/inbox/chat-box.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                                <span key={id++} className="p-3 border rounded-[8px] bg-primary text-white w-fit max-w-[80%]">
+                                    {message}
+                                </span>
+                                </div>
+                            ))}
+                        </div>
+                    </ScrollArea>
+                    <form onSubmit={handleSend}>
+                        <div className="relative">
+                            <Input
```
Comment (jesuskcapellan) [inline src/app/(user)/inbox/chat-box.tsx:62]: see if you can use the textarea component from shadcn, not sure how it works but could be fruitful???

**Review by jesuskcapellan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**